cp-quickstart/start-docker-cloud.sh

```diff
@@ -5,95 +5,169 @@
 # To avoid unexpected charges, carefully evaluate the cost of resources before launching the script and ensure all resources are destroyed after you are done running it.
 #########################################
 
+NAME=`basename "$0"`
+LOG_FILE="$NAME.log"
+
+printf "\n>>>>>>>>>> starting %s @ %s >>>>>>>>>>\n\n" $NAME "$(date)" >> "$LOG_FILE"
 
 # Source library
-. ../utils/helper.sh
+source ../utils/helper.sh
+
+check_jq \
+  && print_pass "jq found"
 
 check_ccloud_version 1.0.0 \
-  && print_pass "ccloud version ok" \
-  || exit 1
+  && print_pass "ccloud version ok"
+
 check_ccloud_logged_in \
-  && print_pass "logged into ccloud CLI" \
-  || exit 1
-
-echo
-wget -O docker-compose.yml https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-cloud/docker-compose.yml
-
-echo ====== Confirm
-prompt_continue_cloud_demo || exit 1
-read -p "Do you acknowledge this script creates a Confluent Cloud KSQL app (hourly charges may apply)? [y/n] " -n 1 -r
-echo
-if [[ ! $REPLY =~ ^[Yy]$ ]]; then exit 1; fi
-
-echo
-echo ====== Create new Confluent Cloud stack
-cloud_create_demo_stack true || exit 1
-SERVICE_ACCOUNT_ID=$(ccloud kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4;}')
+  && print_pass "logged into ccloud CLI" 
+
+print_pass "Prerequisite check pass"
+
+[[ -z "$AUTO" ]] && {
+  printf "\n====== Confirm\n\n"
+  prompt_continue_cloud_demo || exit 1
+  read -p "Do you acknowledge this script creates a Confluent Cloud KSQL app (hourly charges may apply)? [y/n] " -n 1 -r
+  if [[ ! $REPLY =~ ^[Yy]$ ]]; then exit 1; fi
+	printf "\n"
+} 
+
+printf "\nThis demo writes a log file located at:\n\t%s\n" "$(cd $(dirname $LOG_FILE);echo $PWD)/$LOG_FILE"
+printf "\nFor your reference the demo will highlight some commands in "; print_code "code format"
+printf "\nFor example:\n"
+print_code_pass -c "successful command";
+printf "\n"
+print_code_error -c "failed command" -m "failed command message"
+
+printf "\n====== Starting\n\n"
+
+(wget -q -O docker-compose.yml https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-cloud/docker-compose.yml | tee -a $LOG_FILE) \
+  && print_pass "retrieved docker-compose.yml from https://github.com/confluentinc/cp-all-in-one/cp-all-in-one-cloud/docker-compose.yml" \
```

Comment:
```suggestion
  && print_pass "retrieved docker-compose.yml from https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-cloud/docker-compose.yml" \
```

Reply:
@rspurgeon just curious if you missed this suggestion or didn't want to make this change?  If the latter, why not?  The concern would be that the message doesn't reflect the exact URL used.

Reply:
I didn't make the message match the download link intentionally, choosing instead to use the GH URL so a human 'browser' would land on the GH rendered page instead of the raw file.  This particular printout isn't the exact instruction ran by the script, instead I choose to give the reader a general message here.

Reply:
```
✔ retrieved docker-compose.yml from https://github.com/confluentinc/cp-all-in-one/cp-all-in-one-cloud/docker-compose.yml
```

This link doesn't resolve

Reply:
doh, thank you... fixed